src/docs/stan-reference/functions.tex

```diff
@@ -494,7 +494,13 @@ \subsection{Logical Functions}
       x & \mbox{if } cond \neq 0 \\
       y & \mbox{otherwise}
     \end{cases}
-    \]}
+    \]
+    {\bfseries Warning:} This function should not be used.  Rather,
```

Comment:
I think we should deprecate `if_else` with this pull request. I don't see why we should warn users in the doc without warning them at compile time.